client/blocks/stats-navigation/index.js

```diff
@@ -95,13 +95,19 @@ class StatsNavigation extends Component {
 			( toggleItem ) => {
 				// disable the "videos" toggle on sites that do not have VideoPress enabled
 				// the toggle will be disabled (grayed out and non interactive)
-				const shouldDisableVideoToggle = ! nextProps.hasVideoPress && toggleItem.key === 'videos';
+				// TODO: Review this logic.
+				// By default, all sites have the ability to upload a single video.
+				// Therefore, it probably doesn't make sense to disable the module by default.
+				const optionallyDisableVideoPress = false;
+				if ( toggleItem.key === 'videos' && optionallyDisableVideoPress ) {
+					return {
+						...toggleItem,
+						disabled: ! nextProps.hasVideoPress, // This value is currently ignored.
+						defaultValue: nextProps.hasVideoPress,
+					};
+				}
 
-				return {
```

Comment:
Previous logic here was being applied to every `toggleItem` which made it impossible to have an item disabled by default. New logic only updates state for the `videos` item. Currently disabled based on the `optionallyDisableVideoPress ` boolean value.